exec/java-exec/src/test/java/org/apache/drill/exec/store/parquet/TestParquetMetadataCache.java

```diff
@@ -207,6 +207,31 @@ public void testAbsentPluginOrWorkspaceError() throws Exception {
         .go();
   }
 
+  @Test //DRILL-4511
+  public void testTableDoesNotExistError() throws Exception {
```

Comment:
1. Please split into two tests.
2. You can use to `ExecTest.getTempDir()` to get unique temporary directory.